scripts/artifacts-building/containers/build-process.sh

```diff
@@ -66,8 +66,24 @@ sed -i.bak '/lxc_cache_download_template_extra_options: /d' /etc/openstack_deplo
 sed -i.bak '/lxc_container_variant: /d' /etc/openstack_deploy/user_osa_variables_defaults.yml
 sed -i.bak '/lxc_container_download_template_extra_options: /d' /etc/openstack_deploy/user_osa_variables_defaults.yml
 
-# Set override vars for the artifact build
+# TODO(odyssey4me):
+# Remove this once we have published artifacts
+# for a real tagged version.
+#
+# For the script which guesses the version to work
+# it requires the checkout to be a local branch
+# with a name of the form:
+#   <whatever>-<major version>.<minor version>
+# We therefore ensure that we have such a branch so
+# that we can test PR's against the artifacts-14.0
+# branch.
+#
+if ! git branch -v | grep -q ' artifacts-14.0 '; then
```

Comment:
If the artifacts-14.0 branch does already exist, do you want to check it out? Maybe rm the branch ignoring failure, then always checkout -b?